### PR TITLE
[BIA-446] Update release build process

### PR DESCRIPTION
**Pull request by jleiva-gap**
Update TagAndRelease.ps1 : add the LICENSE and NOTICES.md files to the binary release zip files.

### CODE REVIEW

**Comment by DavidJGapCR** [comment]
Hi @stephenfuqua. Can you give us a hand on this. I am not sure how I can test it. Thanks.

**Comment by stephenfuqua** [comment]
> Hi @stephenfuqua. Can you give us a hand on this. I am not sure how I can test it. Thanks.

Sorry, I only just now saw this message. Taking a look now.

**Comment by stephenfuqua** [comment]
@jleiva-gap @DavidJGapCR  Clone the AMT repo and run the script against your clone

File: TagAndRelease.ps1 [hunk 1]
```diff
@@ -44,13 +45,19 @@ Try {
...
+	$uploadUrlFdd = $uploadUrl.replace("{?name,label}", "?name=$fddZipFile");
     $result = Invoke-RestMethod -Method POST -Uri "$uploadUrlFdd" -Headers $headers -Infile "$publishDirectory\$fddZipFile";
     Write-Host $result;
 
+	# Add NOTICES and License files to Self-contained deployment zip file.
+	Compress-Archive -Path $licenseFile, $noticesFile -Update  -DestinationPath $scdZipFile
+	
     # Upload Self-contained deployment zip file
-    $uploadUrlScd = $uploadUrl.replace("{?name,label}", "?name=$scdZipFile");
+	$uploadUrlScd = $uploadUrl.replace("{?name,label}", "?name=$scdZipFile");
```
Comment (stephenfuqua) [inline TagAndRelease.ps1:60]: Minor issue - can you fix the spacing?

File: TagAndRelease.ps1 [hunk 2]
```diff
@@ -44,13 +45,19 @@ Try {
     $uploadUrl = $result.upload_url;
     $headers.'Content-Type' = "application/zip";
 
-    # Upload Framework-dependent deployment zip file
-    $uploadUrlFdd = $uploadUrl.replace("{?name,label}", "?name=$fddZipFile");
+    # Add NOTICES and License files to Framework-dependent deployment zip file.
+	Compress-Archive -Path $licenseFile, $noticesFile -Update  -DestinationPath $fddZipFile
```
Comment (stephenfuqua) [inline TagAndRelease.ps1:49]: Looks good here. Except minor spacing issue.

**Review by stephenfuqua — APPROVED** [review]

**Pull request merged**